pvactools/tools/pvacview/ui.R

```diff
@@ -224,25 +287,6 @@ explore_tab <- tabItem(
                         strong("DeepImmuno"), span(": Deep-learning empowered prediction of immunogenic epitopes for T cell immunity. ("),
                         a(href = "https://academic.oup.com/bib/article/22/6/bbab160/6261914?login=false", "Citation"), span(")"),
                         style = "overflow-x: scroll;"
-                    ),
-                    tabPanel("Anchor Heatmap",
-                        h4("Allele specific anchor prediction heatmap for top 20 candidates in peptide table."),
-                        plotOutput(outputId = "peptideFigureLegend", height = "50px"),
-                        plotOutput(outputId = "anchorPlot") %>% withSpinner(color = "#8FCCFA"), style = "overflow-x: scroll;"
-                    )
-                ),
-                box(
-                    column(width = 4,
-                        h4("Allele Specific Anchor Prediction Heatmap"),
-                        h5(" This tab displays HLA allele specific anchor predictions overlaying good-binding peptide sequences generated from each specific transcript.", br(),
-                            " Current version supports the first 15 MT/WT peptide sequence pairs (first 30 rows of the peptide table)."), br(),
-                        h4("MHC Binding Prediction Scores"),
-                        h5(" This tab contains violin plots that showcase individual binding prediction scores from each algorithm used. A solid line is used to represent the median score.")
-                    ),
-                    column(width = 8,
-                        box(title = "Anchor vs Mutation position Scenario Guide", collapsible = TRUE, collapsed = FALSE, width = 12,
-                            img(src = "https://github.com/griffithlab/pVACtools/raw/master/pvactools/tools/pvacview/www/anchor.jpg",
```

Comment:
Was the WT/MT schema meant to be removed?

Reply:
It was moved to a different box: https://github.com/griffithlab/pVACtools/pull/1012/files#diff-73fbe2993d01a23c16329242cc736f6a80d35e261ce9c88d0f5705df110ddea4R235. I felt that the whole anchor heatmap and WT/MT schema made more sense in the Transcript and Peptide Set Data instead of the Additional Peptide Information box because everything else in the Additional Peptide Information box is specific to one selected peptide.